plugins/spi/SPIOutput.cpp

```diff
@@ -689,7 +892,9 @@ RDMResponse *SPIOutput::SetDeviceLabel(const RDMRequest *request) {
 }
 
 RDMResponse *SPIOutput::GetSoftwareVersionLabel(const RDMRequest *request) {
-  return ResponderHelper::GetString(request, string("OLA Version ") + VERSION);
```

Comment:
Did this need changing? Or is this an editor change, which might explain a lot of my other comments.

Reply:
both no - i did it (as i thought its easier to read this way - i will revoke it..)

Reply:
I don't really mind, although compactness often helps given we don't actually care about that line. If it's going to be on multiple lines, we generally live the first argument with the function/method, then indent the rest to match, unless one of the lines is too long. But we normally squeeze a lot onto the line until we hit the line length.